### DISCUSSION TITLE
Tagging and Branching strategy

### DISCUSSION THREAD

**Discussion by nicoburns** [DioxusLabs/taffy#264] [General]
Currently taffy has two tags: `0.1` and `0.2`. I would like to suggest that:

- We make tag versions more specific (`0.1.0` and `0.2.0`)
- We create branches for minor versions (`0.1 and `0.2` branches).

This would enable the possibility of putting out patch versions of taffy (e.g. a `0.2.1` version). I don't think there's any need for taffy to maintain old versions, but it may be useful to put out a patch version to the latest released version without including all of the changes on main. Examples of issues where I think this would make sense:

- #217
- #263
- and possibly #218 (in the case that there was high demand for the feature)

What do people think?

**Comment by Weibye** [comment]
> We make tag versions more specific (0.1.0 and 0.2.0)

This I think makes a lot of sense :)

> We create branches for minor versions (0.1 and 0.2` branches).

Do we need to keep branches around before there's an actual need for a patch release? Could it be enough to tag a commit with `0.2.0` then create a branch from that point if we see there's a need for a `2.0.1`, from which point you keep the branch alive.

**Reply by alice-i-cecile** [reply]
> Do we need to keep branches around before there's an actual need for a patch release

I agree here :)

**Comment by alice-i-cecile** [comment]
Generally on board here! See @Weibye's suggestion though. @nicoburns you've done an incredible job with the last release.

I've given you Maintainer permissions for this repo: feel free to implement this.

**Comment by alice-i-cecile** [comment]
I've enabled branch protection for the 0.x.* branches too now :)
